app/recordtransfer/templates/registration/login.html

```diff
@@ -13,7 +13,7 @@
             <!-- Form -->
             <div id="login-form-container">
                 <form class="space-y-4"
-                      hx-post="{% url 'login' %}"
+                      hx-post="{% url 'login' %}?next={{ next }}"
```

Comment:
Just a question, `{{ next }}` may be empty, does it matter to worry about that case? I see it re-directs just fine even when next is in the parameters but is empty, just wondering if you think we need to consider that at all.

We could do something like this:

```suggestion
                      hx-post="{% url 'login' %}{% if next }?next={{ next }}{% endif %}"
```

Reply:
I think it wouldn't hurt to do it this way to avoid any unexpected issues.